README.md

```diff
@@ -2,21 +2,30 @@
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/probot/autolabeler.svg)](https://greenkeeper.io/)
 
-> a GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
+> A GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
 
 ## Using
 
-Configure by creating a `.github/autolabeler.yml` file with a yaml file in the format of `label: file/path`. 
+Configure by creating a `.github/autolabeler.yml` file with a [yaml file](https://en.wikipedia.org/wiki/YAML) in the format of `label: file/path`. Then add the [Probot Auto Labeler Bot](https://github.com/apps/probot-autolabeler) to your repository.
```

Comment:
would you mind turning `yaml` to uppercase `YAML`?

Reply:
Fixed.